app/src/main/java/io/homeassistant/companion/android/settings/views/SettingsRow.kt

```diff
@@ -0,0 +1,74 @@
+package io.homeassistant.companion.android.settings.views
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.unit.dp
+import com.mikepenz.iconics.compose.Image
+import com.mikepenz.iconics.typeface.IIcon
+
+@Composable
+fun SettingsRow(
+    primaryText: String,
+    secondaryText: String,
+    mdiIcon: IIcon?,
+    enabled: Boolean,
+    onSensorClicked: () -> Unit
```

Comment:
Because this is now a generic composable, maybe change the parameter name to be generic as well?

Reply:
Thanks!